airflow/providers/amazon/aws/hooks/emr.py

```diff
@@ -36,7 +38,11 @@ def __init__(self, emr_conn_id=None, *args, **kwargs):
         self.emr_conn_id = emr_conn_id
         super().__init__(client_type='emr', *args, **kwargs)
 
-    def get_cluster_id_by_name(self, emr_cluster_name, cluster_states):
+    def get_cluster_id_by_name(
+        self,
+        emr_cluster_name: str,
+        cluster_states: Optional[List[str]]
```

Comment:
Probably this should be:
```python
cluster_states: List[str]
```
However, if it's really optional, then the correct use would be (keep in mind this would change the original function's signature):
```python
cluster_states: Optional[List[str]] = None
```

Reply:
The `cluster_states` is defined as Optional in `emr_add_steps.py` where the function is used.
Is it correct that the original function's signature is changed?

Reply:
Thanks, I rechecked the boto3 API and you were right.
Also changed the `cluster_states` in `emr_add_steps.py`.